static/app/components/events/highlights/util.tsx

```diff
@@ -170,3 +171,38 @@ export function getHighlightTagData({
     originalTag: tagMap[tagKey]?.tag ?? {key: tagKey, value: EMPTY_HIGHLIGHT_DEFAULT},
   }));
 }
+
+/**
+ * Returns a label that can be used in the Event Highlight Summary.
+ * Currently only used for JavaScript-based events.
+ */
+export function getRuntimeLabelAndTooltip(
+  event: Event,
+  knownRuntimeType?: {isBackend?: boolean; isFrontend?: boolean}
```

Comment:
this is a bit of a weird API, what if you pass `{ isBackend: true, isFrontend: true }`? I'd rather make this an enum:

```js
knownRuntimeType?: 'frontend' | 'backend'
```